bootstrap.d/11-mod.sh

```diff
@@ -36,7 +36,6 @@ wget https://activities.sugarlabs.org/en-US/sugar/downloads/file/29282/stopwatch
 wget https://people.sugarlabs.org/rishabh/Physics-32.1.xo -P files/Sugar-activities
 wget https://people.sugarlabs.org/rishabh/TurtleBlocks-216.xo -P files/Sugar-activities
 
-
```

Comment:
- [ ] no reason given for removing this blank line, perhaps it wasn't intentional, can't tell..

Reply:
I don't think an extra blank line would affect the way this script executes, I can add it if it's necessary though.

Reply:
Nothing to do with the way the script executes; it will just take a little bit longer with a blank line than without, but the amount of time is microscopic.  My review comment is that you've removed a blank line, you didn't say you'd done it, so it makes it look like you don't know what you are changing.  Changes unrelated to the purpose of a commit make `git bisect` very hard to use, and I'm a heavy user of `git bisect`, so I'm sensitive to unrelated changes.  If you can justify the change as necessary (e.g. an extra blank line was added by accident in a previous commit), then you can also explain the change in the commit description.

Reply:
I understand your concern, it was added accidently in the previous commit, I realised this while making changes to the file.